core/renderers/measurables/connections.js

```diff
@@ -47,7 +47,7 @@ Blockly.blockRendering.Connection = function(constants, connectionModel) {
       constants);
   this.connectionModel = connectionModel;
   this.shape = this.constants_.shapeFor(connectionModel);
-  this.isDynamicShape = !!this.shape.isDynamic;
+  this.isDynamicShape = !!this.shape['isDynamic'];
```

Comment:
Why the change in style?

Reply:
shapeFor is returning an object, we don't have any idea of its "shape" (no pun intended). 
When we use an interface for shapes this will go away.